pkg/controller.v2/controller_pod.go

```diff
@@ -168,14 +173,14 @@ func (tc *TFJobController) createNewPod(tfjob *tfv1alpha2.TFJob, rt, index strin
 		})
 	}
 
-	if spec.RestartPolicy == tfv1alpha2.RestartPolicyExitCode {
-		podTemplate.Spec.RestartPolicy = v1.RestartPolicyNever
-	} else if spec.RestartPolicy == tfv1alpha2.RestartPolicy("") {
-		// Set default to Never.
-		podTemplate.Spec.RestartPolicy = v1.RestartPolicyNever
-	} else {
-		podTemplate.Spec.RestartPolicy = v1.RestartPolicy(spec.RestartPolicy)
+	// Submit a warning event if the user specifies restart policy for
+	// the pod template. We recommend to set it from the replica level.
+	if podTemplate.Spec.RestartPolicy != v1.RestartPolicy("") {
+		errMsg := "Restart policy in pod template may be override by restart policy in replica spec"
```

Comment:
Can we say definitively whether or not the Restart policy will be overwritten?

Also grammer issue

"may be override" -> "may be overwritten"

Reply:
You could say something like "User defined Pod RestartPolicy will be ignored; pod RestartPolicy will be set as necessary based on Replica RestartPolicy. To avoid this warning don't set RestartPolicy for the pod."

Reply:
We could say that `Restart policy in pod template will be overwritten by restart policy in replica spec`. There is only one case that the policy will not be overwritten, it is triggered when the policy defined in pod template is the same as replica spec. And it means the policy is overwritten by the same policy.

Reply:
I will update it now.